src/expr/tconv_seq_proof_generator.cpp

```diff
@@ -0,0 +1,163 @@
+/*********************                                                        */
+/*! \file tconv_seq_proof_generator.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
```

Comment:
I'm not sure what happened but I have noticed a lot of this open parenthesis in headers. Delete it.

Reply:
This is leftover from a bug in the old copyright header.